src/chapters/exploratory-data-analysis/eda.rst

```diff
@@ -1,37 +1,45 @@
 Exploratory Data Analysis
 =========================
 
-Readings
---------
-
 Read the following articles, follow along where instructed:
 
 .. admonition:: tip
   
   For Medium articles: if you run out of free articles, open the page in an incognito window
 
+Exploring Data with Python
+--------------------------
 
-| `Exploratory data analysis in Python <https://towardsdatascience.com/exploratory-data-analysis-in-python-c9a77dfa39ce>`__.
+1. `Exploratory data analysis in Python <https://towardsdatascience.com/exploratory-data-analysis-in-python-c9a77dfa39ce>`__.
 
 * Key Takeaways: Outline of common steps used in EDA, note that there is no one process for performing EDA, 
   it all depends on the dataset and your questions.
 * Just read and follow the steps for comprehension, don't need to do the tutorial.
 * Stop at #8: “Detecting Outliers”
-  
-| `Getting to know your data <https://medium.com/@shanegary/getting-to-know-your-data-9e42935e7f60>`__.
+
+Get to Know Your Data
+---------------------
+
+2. `Getting to know your data <https://medium.com/@shanegary/getting-to-know-your-data-9e42935e7f60>`__.
 
```

Comment:
Under each new section ordered lists should restart and not continue from the previous section.

Use #. in front of link insted of hard coding.